packages/@uppy/provider-views/src/CloseWrapper.js

```diff
@@ -1,4 +1,4 @@
-const { h, Component } = require('preact')
+const { Component } = require('preact')
```

Comment:
Just FYI, I had to remove this unused import because Travis/`npm run lint` were complaining.